IPython/core/shellapp.py

```diff
@@ -139,7 +139,18 @@ class InteractiveShellApp(Configurable):
     extra_extension = Unicode('', config=True,
         help="dotted module name of an IPython extension to load."
     )
-    
+
+    def _extra_extension_changed(self, name, old, new):
```

Comment:
Will fix this in the morning, looks like I accidentally reverted 34a0e6b8867a4fd448c721d006e8253e8cec1089